StarterProject/README.md

```diff
@@ -0,0 +1,83 @@
+
+****************************************************************************************************************
+
+STARTER PROJECT
+
+
+This starter project is used to give you the user a short introduction into webdriver.io and to help you to get up and running as soon as possible.
+
+This starter project contains a suite of tests that accomplishes the following actions:
+ - Navigate to http://www.clubmonaco.ca/
+ - Check if the email modal exists
+ - Close the email modal
+ - Search for a prouct with the search
+ - Ensure that related product flyout is present
+ - Navigate to a product page
+ - View the assets in the product page
+
+****************************************************************************************************************
+
+PREREQUISITES
+
+
+ Encountered a problem with reinstallig webdriver.io with npm install so this workaround of installing global packages instead.
+ Install all prerequisites with ./install-global-npm.sh 
```

Comment:
Maybe we can resolve this, there shouldn't be any issues just installing it locally either with yarn or npm